pixi.toml

```
@@ -0,0 +1,18 @@
+[project]
+name = "rattler"
+version = "0.7.0"
+description = "Rust library to install conda environments"
+authors = ["Wolf Vollprecht <wolf@prefix.dev>", "Bas Zalmstra <bas@prefix.dev>", "Tim de Jager <tim@prefix.dev>", "Ruben Arts <ruben@prefix.dev>"]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
```

Please also include additional platforms:

```suggestion
platforms = ["linux-64", "win-64", "osx-64", "osx-arm64"]
```